test/integration/members.test.js

```diff
@@ -289,4 +292,54 @@ describe("Members", function () {
       });
     });
   });
+
+  describe("POST /members/cache/clear/self", function () {
+    afterEach(function () {
+      sinon.restore();
+    });
+
+    it("Should purge the cache of member's profile page", function (done) {
+      sinon.stub(members, "purgeMembersCache").returns(purgeCache[0]);
+      chai
+        .request(app)
+        .post("/members/cache/clear/self")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Cache purged successfully");
+          expect(res.body.success).to.equal(true);
+          expect(res.body.errors).to.deep.equal([]);
+          expect(res.body.messages).to.deep.equal([]);
+          expect(res.body.result).to.be.a("object");
+          expect(res.body.result.id).to.be.a("string");
+
+          return done();
+        });
+    });
+
```

Comment:
For Discord Discussion - https://discord.com/channels/673083527624916993/1000471526174359663